server/v1/models/tripModel.js

```diff
@@ -0,0 +1,100 @@
+import moment from 'moment';
+
+class Trip {
+  /**
+  * class constructor
+  * @param {object} data
+  */
+  constructor() {
+    this.trips = [
+      {
+        id: 1,
+        seating_capacity: 45,
```

Comment:
Avoid hardcoding data/content in the models

Reply:
Done